entity-management-tests/src/integration-test/java/eu/europeana/entitymanagement/AbstractIntegrationTest.java

```diff
@@ -276,7 +279,9 @@ protected EntityRecord createEntity(
             europeanaProxyEntity, xmlBaseEntity.toEntityModel(), dataSource, null);
 
     // trigger update to generate consolidated entity
-    entityUpdateService.runSynchronousUpdate(savedRecord.getEntityId());
+    entityUpdateService.runSynchronousUpdate(savedRecord.getEntityId(),
+            Arrays.asList(Task.DEREFERENCE, Task.CONSOLIDATION, Task.METRICS, Task.VALIDATION),
```

Comment:
As the list of the tasks and writers are common for many methods, we can have these lists defined as static variables e.g. PROCESSORS_FULL_UPDATE,  PERSISTENCE_ITEM_WRITERS